mailbox/api/src/main/java/org/apache/james/mailbox/model/UpdatedFlags.java

```diff
@@ -104,7 +143,19 @@ private static boolean isChanged(Flags original, Flags updated, String userFlag)
     public Flags getOldFlags() {
         return oldFlags;
     }
-    
+
+    public boolean isSet(Flags.Flag flag) {
```

Comment:
I agree with @mbaechler maybe hasJustBeenSet would be a better name. But I am bad at naming stuff that speak to other people than myself.